apps/src/blockly/addons/cdoCss.ts

```diff
@@ -76,6 +76,9 @@ export default function initializeCss(blocklyWrapper: BlocklyWrapperType) {
       height: 35px;
       width: 45px;
     }
+    .blocklyDropDownDiv .blocklyMenu {
+      font-family: ${fontConstants['main-font']};
```

Comment:
I recommend enforcing `font-weight: 400` to ensure the issue doesn't recur.
```suggestion
      font-family: ${fontConstants['main-font']};
      font-weight: 400 !important; // Noto Sans Math only supports the normal font-weight
```